source/extensions/omni.isaac.lab/omni/isaac/lab/utils/math.py

```diff
@@ -1059,6 +1062,100 @@ def unproject_depth(depth: torch.Tensor, intrinsics: torch.Tensor) -> torch.Tens
     return points_xyz
 
 
+@torch.jit.script
+def convert_perspective_depth_image_to_orthogonal_depth_image(
```

Comment:
It's clear that this function will be called for perspective depth. Good to avoid very long function names.

```suggestion
def convert_depth_image_to_orthogonal(
```

Reply:
I shortened this method name to be 

`convert_perspective_depth_to_orthogonal_depth`

I'd make the case that it is important to include that the type of depth image being converted is perspective. While this may be evident from the doc string, I think having just `convert_depth_image_to_orthogonal` could lead to confusion, especially when `unproject_depth` only works correctly with an orthogonal depth image.